docs/networkpolicy-metrics.md

```diff
@@ -0,0 +1,9 @@
+# Network Policy Metrics
+
+
+| Metric name                           | Metric type | Labels/tags                                 | Status       |
+| ------------------------------------- | ----------- | ------------------------------------------- | ------------ |
+| kube_networkpolicy_created            | Gauge       | `networkpolicy`=&lt;networkpolicy name&gt;  | EXPERIMENTAL |
```

Comment:
These should be `STABLE` imo. They are all well-established fields in NetworkPolicy.

Reply:
👍 I wasn't sure what the convention was

Reply:
@tariq1890 See my open issue :D https://github.com/kubernetes/kube-state-metrics/issues/892

@milesbxf sorry about this, we are still discussing, this we need to also document it better! :)

Reply:
@LiliC Thanks for pointing that out. I think we can agree that these docs are fine as-is. So I'm resolving.